common.blocks/control-group/control-group.bemhtml

```diff
@@ -0,0 +1,6 @@
+block('control-group').attrs()(function() {
+    return {
+        role : 'group',
+        'aria-label' : this.ctx.ariaLabel
```

Comment:
Remove `ariaLabel` special field from public API. Add `aria-label="."` attribute instead. JAWS should read this control as "group start / group end". User can specify `aria-label` or `aria-labelledby` attributes via `attrs` field in BEMJSON.